CHANGELOG.md

```diff
@@ -20,9 +20,11 @@ Current Trunk
   - `i32.wait` / `i64.wait` -> `i32.atomic.wait` / `i64.atomic.wait`
 - Binaryen.js: `flags` argument in `setMemory` function is removed.
 - `atomic.fence` instruction support is added.
-- wasm-emscripten-finalize: Don't realy on name section being present in the
+- wasm-emscripten-finalize: Don't rely on name section being present in the
   input. Use the exported names for things instead.
 - Added `mutable` parameter to BinaryenAddGlobalImport.
+- Replace BinarySIMDBitselect* with BinarnSIMDTernary* in the C API and add
```

Comment:
```suggestion
- Replace BinarySIMDBitselect* with BinarySIMDTernary* in the C API and add
```

Reply:
Wow this is all kinds of wrong. Thanks for the catch!

Reply:
Oh `Binaryen*` makes much more sense :)

Reply:
WDYT switch to more general name `makeTernary`? According to [FutureFeatures.md](https://github.com/WebAssembly/design/blob/master/FutureFeatures.md#additional-floating-point-operators) WebAssembly probably introduce fused multiply add instructions for scalar as well or only SIMD planning?

Reply:
I agree that renaming to `Ternary` would make sense if we ever get ternary scalar operations. We could even rename `Select` to `Ternary` and getting rid of the separate `SIMDTernary` altogether, which would be consistent with how we treat SIMD unary and binary ops. I don't know of any concrete plans to add new scalar ternary operations, though, and I think keeping the separate SIMD instruction type is useful for now because it simplifies the common case in which SIMD is not considered.